OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -814,12 +811,12 @@ public void ResolveOrder(Actor self, Order order)
 			else if (order.OrderString == "ReturnToBase" && rearmableInfo != null && rearmableInfo.RearmActors.Any())
 			{
 				if (!order.Queued)
+				{
 					UnReserve();
+					self.CancelActivity();
```

Comment:
This one can be removed now that order.Queued is already forwarded to QueueActivity.